hstream-admin-server/app/Main.hs

```diff
@@ -0,0 +1,89 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards   #-}
+{-# LANGUAGE TypeOperators     #-}
+
+module Main where
+
+import           Control.Exception              (catch)
+import           Control.Monad                  (void)
+import           Control.Monad.IO.Class         (liftIO)
+import           Data.ByteString                (ByteString)
+import           Network.Wai.Handler.Warp       (run)
+import           Options.Applicative            (Parser, auto, execParser,
+                                                 fullDesc, help, helper, info,
+                                                 long, metavar, option,
+                                                 progDesc, short, showDefault,
+                                                 strOption, value, (<**>))
+import           Servant                        (Proxy (..), (:<|>) (..))
+import           Servant.Server                 (Server, serve)
+import qualified Z.Data.CBytes                  as ZDC
+import qualified ZooKeeper                      as ZK
+import qualified ZooKeeper.Exception            as ZK
+import qualified ZooKeeper.Types                as ZK
+
+import           HStream.Admin.Server.Connector (ConnectorsAPI, connectorServer)
+import           HStream.Admin.Server.Node      (NodesAPI, nodeServer)
+import           HStream.Admin.Server.Overview  (OverviewAPI, overviewServer)
+import           HStream.Admin.Server.Query     (QueriesAPI, queryServer)
+import           HStream.Admin.Server.Stream    (StreamsAPI, streamServer)
+import           HStream.Server.Persistence     as HSP
```

Comment:
this should be:

`import     qualified      HStream.Server.Persistence     as HSP`

Reply:
done